CMakeLists.txt

```diff
@@ -24,19 +24,17 @@
 
 cmake_minimum_required(VERSION 3.2)
 include(CheckCXXSymbolExists)
-project (sqlpp11-connector-sqlite3)
+project (sqlpp11-connector-sqlite3 VERSION 0.28 LANGUAGES CXX)
```

Comment:
Can you think of a way to avoid putting the version number here?

Versions are tagged in git. I would not want to have to keep the CMakeLists in sync (I will forget to do that for sure).

Reply:
At first I tried to mimick what was in sqlpp11 CMakeLists.txt. But the version was 0.1.

I removed the version file in the end.